composer.json

```diff
@@ -14,23 +14,24 @@
     "require": {
         "php": ">=7.2",
         "doctrine/doctrine-bundle": "^1.8|^2.0",
-        "doctrine/orm": "^2.6",
+        "doctrine/orm": "^2.7|^3.0",
         "league/oauth2-server": "^8.0",
         "psr/http-factory": "^1.0",
-        "sensio/framework-extra-bundle": "^5.3",
-        "symfony/framework-bundle": "^3.4|^4.2",
-        "symfony/psr-http-message-bridge": "^1.2",
-        "symfony/security-bundle": "^3.4|^4.2"
+        "sensio/framework-extra-bundle": "^5.5",
+        "symfony/psr-http-message-bridge": "^1.3",
+        "symfony/security-bundle": "^4.4|^5.0",
+        "symfony/framework-bundle": "^4.4|^5.0.2"
     },
     "require-dev": {
         "ext-timecop": "*",
         "ext-xdebug": "*",
-        "friendsofphp/php-cs-fixer": "2.16.0",
-        "nyholm/psr7": "^1.1",
-        "phpunit/phpunit": "^8.4",
-        "symfony/browser-kit": "^3.4|^4.2",
-        "symfony/phpunit-bridge": "^3.4|^4.4",
-        "zendframework/zend-diactoros": "^2.1"
+        "friendsofphp/php-cs-fixer": "^2.16",
+        "laminas/laminas-diactoros": "^2.2",
+        "nyholm/psr7": "^1.2",
+        "phpunit/phpunit": "^8.5",
+        "roave/security-advisories": "dev-master",
```

Comment:
This addition is unrelated to Symfony 5.

Reply:
- nyholm/psr7: minor point upgrade from master 1.1 to 1.2, i think we should keep to stay up to date in dev (should keep)
- phpunit: minor point upgrade from master 8.4 to 8.5, i think we should keep to stay up to date in dev (should keep)
- laminas: replaces the deprecated zendframework/zend-diactoros package, which would keep throwing warnings in test. I upgraded all tests to reference laminas instead of zendframework (must keep)
- i do think that we should also keep roave/security-advisories in dev because it will help us flag packages with security issues during development. But I can remove it if you really wish. (i can remove)

Reply:
I was talking only about `roave/security-advisories` here. I'd like to keep this PR as minimal as possible and just focus on Symfony 5 support. Stuff like `roave/security-advisories`  can be PR-ed and considered separately. The bumping of other dev dependencies is fine.

Reply:
ok I removed roave/security-advisories